saleor/graphql/csv/enums.py

```diff
@@ -0,0 +1,13 @@
+import graphene
+
+from ...csv import JobStatus
+from ...graphql.core.enums import to_enum
+
+
+class ExportScope(graphene.Enum):
```

Comment:
We should provide descriptions for these enums so that it's clear in the API what's the difference between these three export modes.